### DISCUSSION TITLE
Membership for officials

### DISCUSSION THREAD

**Discussion by endwaa** [jflamy/owlcms4#887] [Ideas]
In the athlete registration sheet there is a field for adding membership, which is a good and useful feature. But is there any plans to add a similare field for officials? 

The Norwegian Weightlifting Federation is working on a solution to save information from a competition into our own system as automatically as possible. The system require an unqiue identification for both athletes and officials, and a field for official membership would be really helpful to automate the process further.

**Comment by jflamy** [comment]
Step 1 could be just to create a table for the officials, with their membership, affiliation, and level.
Step 2 could be to enforce the fact that the referee comes from the table.
Step 3 could be adding whether the TO is on a referee team or a jury team.  IWF tends to assign referees for one session on, one session off, whereas jurys are assigned two sessions in a row so it's not always the same jury with the same referees.

Please create a feature request on the page https://github.com/jflamy/owlcms4/issues -- this way you will be informed of eventual progress when I start working on this.  I have no way to add you if I create the feature request.

**Reply by endwaa** [reply]
Sounds good 👍
